packages/block-library/src/navigation/edit.js

```diff
@@ -84,6 +86,14 @@ function Navigation( {
 	//
 	// HANDLERS
 	//
+	const handleItemsAlignment = ( align ) => {
```

Comment:
For the sake of better debugging (since anonymous functions like this one lack names in stack traces), I think this should be changed to a standard function declaration. Writing it as an anonymous function assigned to a constant variable doesn't save any characters anyway.

There are other anonymous functions that could be changed to function declarations below this one, but I'll leave it up to you whether to change those in this PR or not.

Reply:
I understand. I see this pattern a lot across the codebase so I'd be interested to hear @youknowriad's opinion on whether it should be a code style thing to avoid anon functions. 

I've converted them for now.